frontend/src/scenes/session-recordings/player/sidebar/PlayerSidebarEditPinnedPropertiesPopover.tsx

```diff
@@ -0,0 +1,101 @@
+}
+
+export function PlayerSidebarEditPinnedPropertiesPopover(
+    props: PlayerSidebarEditPinnedPropertiesPopoverProps
+): JSX.Element | null {
+    const { loadPerson, loadPersonUUID } = useActions(personsLogic({ syncWithUrl: false }))
+    const { person, personLoading } = useValues(personsLogic({ syncWithUrl: false }))
+    const { logicProps } = useValues(sessionRecordingPlayerLogic)
+    const { allOverviewItems, pinnedProperties } = useValues(playerMetaLogic(logicProps))
+    const { togglePropertyPin } = useActions(playerMetaLogic(logicProps))
+
+    useEffect(() => {
+        if (props.distinctId) {
+            loadPerson(props.distinctId)
+        } else if (props.personId) {
+            loadPersonUUID(props.personId)
+        }
+    }, [loadPerson, loadPersonUUID, props.distinctId, props.personId])
+
+    if (!props.distinctId && !props.personId) {
+        return null
+    }
+
+    if (personLoading) {
+        return <Spinner />
+    }
+
+    // NOTE: This can happen if the Person was deleted or the events associated with the distinct_id had person processing disabled
+    if (!person) {
+        return (
+            <div className="p-2 max-w-160">
+                <h4>No person profile associated with this ID</h4>
+                <p>
+                    Person profiles allow you to see a detailed view of a Person's user properties, track users across
+                    devices, and more. To create person profiles, see{' '}
+                    <Link to="https://posthog.com/docs/data/persons#capturing-person-profiles">here.</Link>
+                </p>
+                <p>To edit pinned overview properties, try via another recording with a different user.</p>
+            </div>
+        )
+    }
+
+    const personPropertyKeys = person.properties ? Object.keys(person.properties).sort() : []
+    const recordingPropertyKeys = allOverviewItems.map((item) =>
+        item.type === 'property' ? item.property : item.label
+    )
+
+    // Deduplicate and create a combined list
+    const allPropertyKeys = Array.from(new Set([...recordingPropertyKeys, ...personPropertyKeys])).sort()
+
+    return (
+        <div className="flex flex-col overflow-hidden max-h-96 w-[400px]">
+            <div className="flex items-center gap-2 px-4 py-3 border-b">
+                <IconPinFilled className="text-muted" />
+                <h4 className="font-semibold m-0">Pinned properties</h4>
+            </div>
+
+            <ScrollableShadows direction="vertical" className="flex-1">
+                <div className="flex flex-col">
+                    {allPropertyKeys.map((propertyKey) => {
+                        const isPinned = pinnedProperties.includes(propertyKey)
+
+                        const handleToggle = (): void => {
+                            togglePropertyPin(propertyKey)
+                        }
+
+                        return (
+                            <LemonButton
+                                key={propertyKey}
```

Comment:
```suggestion
                                key={propertyKey}
                                data-attr={isPinned ? 'session-overview-unpin-property' : 'session-overview-pin-property' }
```
this makes it easy to create an action so we can track whether folk interact here